lib/rspecq/reporter.rb

```diff
@@ -134,6 +139,7 @@ def flaky_jobs_to_sentry(jobs, build_duration)
           object: inspect,
           pid: Process.pid,
           job_path: job,
+          seed: RSpec.configuration.seed,
```

Comment:
Is that correct? The reporter is a different process than the worker that executed the job, so their `RSpec.configuration.seed` values are different. No?

(Note: the change I proposed in the test will answer this for us.)

Reply:
This is true. The seed is probably wrong. I will beef up the tests and this will be surfaced as you mentioned.